ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/cartproductdisplay.html

```diff
@@ -46,7 +46,7 @@
             <span class = 'delivery-charges'>160</span>
 			</div>
         </div>
-		 <div>
+		 <div style = "font-weight: bold">
```

Comment:
@DeepaliAgrwl : we should not be using inlince CSS . Please use CSS heirachy or add a class

Reply:
Comment incorporated